www/grpc/proto/network.proto

```diff
@@ -7,17 +7,18 @@ option java_package = "pactus.network";
 service Network {
   rpc GetNetworkInfo(GetNetworkInfoRequest) returns (GetNetworkInfoResponse);
   rpc GetNodeInfo(GetNodeInfoRequest) returns (GetNodeInfoResponse);
+  rpc GetPeersInfo(GetPeersInfoRequest) returns (GetPeersInfoResponse);
 }
 
 message GetNetworkInfoRequest {}
 
 message GetNetworkInfoResponse {
-  int32 total_sent_bytes = 1;
-  int32 total_received_bytes = 2;
-  int64 started_at = 3;
-  repeated PeerInfo peers = 4;
-  map<int32, int64> sent_bytes = 5;
-  map<int32, int64> received_bytes = 6;
+  int32 protocol_version = 1;
```

Comment:
It is better to add `protocol_version` to the  `GetNodeInfoResponse`

Reply:
Also, the protocol version is available in agent in node info.